container_pipeline/utils.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+
+def get_job_name(job_details):
+    """Get jenkins job name from job_detials"""
+    return str(job_details['appid']) + "-" + str(job_details['jobid']) + "-" \
+        + str(job_details['desired_tag'])
+
+
+class Build:
+    """
+    Track image build status in the pipeline
+
+    In future, we can collate a lot of duplicate code from workers
+    to manage and track build, thereby providing a clean interface
+    to manage builds and prevent duplication of code.
+    """
+
+    def __init__(self, name, datadir='/srv/pipeline-logs'):
+        self.name = name
+        self._path = os.path.join(datadir, name)
+
+    def is_running(self):
```

Comment:
Yes, it's confusing. So, but essentially, the output of this method means whether the pipeline build is running or not. So, in that way, it makes sense from an API consumer point of view.